src/ucp/api/ucp.h

```diff
@@ -262,31 +262,33 @@ enum ucp_ep_params_field {
-                                                           mechanism.
-                                                           @ref ucs_sock_addr_t
-                                                           sockaddr field
-                                                           must be provided and
-                                                           contain the address
-                                                           of the remote peer */
-    UCP_EP_PARAMS_FLAGS_NO_LOOPBACK    = UCS_BIT(1),  /**< Avoid connecting the
-                                                           endpoint to itself when
-                                                           connecting the endpoint
-                                                           to the same worker it
-                                                           was created on.
-                                                           Affects protocols which
-                                                           send to a particular
-                                                           remote endpoint, for
-                                                           example stream */
-    UCP_EP_PARAMS_FLAGS_SEND_CLIENT_ID = UCS_BIT(2)   /**< Send client id
-                                                           when connecting to remote
-                                                           socket address as part of the
-                                                           connection request payload.
-                                                           On the remote side value
-                                                           can be obtained from
-                                                           @ref ucp_conn_request_h using
-                                                           @ref ucp_conn_request_query */
+    /**
+     * Using a client-server connection establishment mechanism.
+     * @ref ucs_sock_addr_t sockaddr field must be provided and contain the
+     * address of the remote peer.
+     */
+    UCP_EP_PARAMS_FLAGS_CLIENT_SERVER  = UCS_BIT(0),
+
+    /**
+     * Avoid connecting the endpoint to itself when connecting the endpoint
+     * to the same worker it was created on. Affects protocols which send to
+     * a particular remote endpoint, for example stream.
+     */
+    UCP_EP_PARAMS_FLAGS_NO_LOOPBACK    = UCS_BIT(1),
+
+    /**
+     * Send client id when connecting to remote socket address as part of the
+     * connection request payload. On the remote side value can be obtained
+     * from @ref ucp_conn_request_h using @ref ucp_conn_request_query.
+     */
+    UCP_EP_PARAMS_FLAGS_SEND_CLIENT_ID = UCS_BIT(2),
+
+    /**
+     * Request selection of transports with capability to perform local
+     * operations using a peer's memory buffer and memory handle associated
+     * with it and created with @ref UCP_MEM_MAP_EXPORTED capability.
```

Comment:
comma after "it" reads better (natural breath point)

Reply:
done